### ISSUE TITLE
train unet problem 

### ISSUE THREAD

**Issue by ljc1231** [LIVIAETS/boundary-loss#31]
i use unet and loss is only generalized dice loss
i found the loss first decrease ,but quickly increase like the picthure below
![image](https://user-images.githubusercontent.com/67781186/101160438-00a24580-366a-11eb-867a-571004fb22c1.png)
i tried trained it more time , finally the loss increase to 1, i don't know why

**Comment by HKervadec** [comment]
What optimizer/learning rate are you using ? Is this with the generalized dice loss alone ?

**Comment by ljc1231** [comment]
i use adam optimizer,and lr i tried from 0.001 to 0.00001. i use ISLES2018 dataset, i divided the original 256*256 pictures into 64*64 patches , i tried GDL, GDL with bundaryloss and ce loss, but it didn't work. 
if i just use original picture 256*256*5, it have train and val dice like 0.7, but the test dice is only 0.3, i don't know how to solve it?

**Comment by HKervadec** [comment]
Closing that one in favor of #30
